man/docker-login.1.md

```diff
@@ -18,9 +18,11 @@ Register or log in to a Docker Registry located on the specified
 do not specify a `SERVER`, the command uses Docker's public registry located at
 `https://registry-1.docker.io/` by default.  To get a username/password for Docker's public registry, create an account on Docker Hub.
 
+`docker login` requires you to use `sudo` or be `root`.
```

Comment:
This is not a true statement, at least not in general across all `docker client` configurations.  I.e. when using docker to connect to a `docker-machine` provisioned `docker engine`.